client/web/compose/src/components/Namespaces/NamespaceSidebar.vue

```diff
@@ -358,7 +364,7 @@ export default {
           },
           children: [],
         },
-        ...this.modules.map(moduleWrap),
+        ...this.modules.map(this.moduleWrap),
```

Comment:
can you just use the old way moduleWrap worked, but provide it with currentPageType aswell